<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crear Usuario...</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>160</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelUser">
      <property name="text">
       <string>Nombre de Usuario</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editUser">
      <property name="statusTip">
       <string>Nombre de Usuario</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelPass">
      <property name="text">
       <string>Contrase単a</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editPass">
      <property name="statusTip">
       <string>Ingrese su Contrase単a</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Repita su contrase単a</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editPass_2">
      <property name="statusTip">
       <string>Ingrese su Contrase単a</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>95</width>
     <height>98</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="btnCrear">
      <property name="text">
       <string>Crear Usuario</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
